src/pages/order-service/OrderServiceControl.tsx

```diff
@@ -125,6 +149,21 @@ function OrderServiceTable() {
     setFilter(query);
   };
 
+  const formattedWorkstations = (data: Workstation[]): ISelectOption[] => {
+    return data?.map((item: Workstation) => {
```

Comment:
Não tem necessidade de dizer qual o tipo de `item`, pois o typescript já entende que é um `Workstation`

```suggestion
    return data?.map((item) => {
```